### PR TITLE
Update analytics_api.py

### PR DESCRIPTION
**Pull request by MatthewPiatetsky**
https://edx-internal.slack.com/archives/G01J7T0CLCS/p1612813879012800

### CODE REVIEW

File: course_discovery/apps/course_metadata/data_loaders/analytics_api.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 class AnalyticsAPIDataLoader(AbstractDataLoader):
 
-    API_TIMEOUT = 120  # time in seconds
+    API_TIMEOUT = 300  # time in seconds
```
Comment (Dillon-Dumesnil) [inline course_discovery/apps/course_metadata/data_loaders/analytics_api.py]: Can you add in a comment about why we are doing this and when we might be able to reduce this back to its current state?

**Review by Dillon-Dumesnil — APPROVED** [review]
Would you mind also updating the PR message with more context. Our slack messages are erased every six months so if someone finds this PR in >6 months, they will be left without any context.

File: course_discovery/apps/course_metadata/data_loaders/analytics_api.py (hunk 1 above)
Comment (mikix) [reply course_discovery/apps/course_metadata/data_loaders/analytics_api.py]: I don't think we plan to reduce it. Brian Mesick looked at the service, didn't see anything particularly wrong with it. Just a lot of data to sift. Mentioned maybe pagination as a possible improvement.

So could add a comment like "Consider reducing this if we ever get pagination" maybe?

**Comment by DawoudSheraz** [comment]
Closing this PR because this is an old PR and there have been no updates on it. It can be re-opened if needed.

**Comment by openedx-webhooks** [comment]
Although this pull request is already closed,  I'm still watching it for updates.

There is nothing you have to do. No action is needed from your side.  Thanks.

**Pull request closed without merging**